WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -3461,6 +3461,9 @@
 		EA14533829AD874C001F3143 /* UITestsFoundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3FA640572670CCD40064401E /* UITestsFoundation.framework */; };
 		EA14533929AD874C001F3143 /* BuildkiteTestCollector in Frameworks */ = {isa = PBXBuildFile; productRef = EA14532529AD874C001F3143 /* BuildkiteTestCollector */; };
 		EA78189427596B2F00554DFA /* ContactUsScreen.swift in Sources */ = {isa = PBXBuildFile; fileRef = EA78189327596B2F00554DFA /* ContactUsScreen.swift */; };
+		EA85B7AB2A686C7A0096E097 /* TestObserver.swift in Sources */ = {isa = PBXBuildFile; fileRef = EA85B7A92A6860370096E097 /* TestObserver.swift */; };
+		EA85B7AF2A688AB00096E097 /* TestObserver.swift in Sources */ = {isa = PBXBuildFile; fileRef = EA85B7A92A6860370096E097 /* TestObserver.swift */; };
+		EA85B7B02A688B1C0096E097 /* TestObserver.swift in Sources */ = {isa = PBXBuildFile; fileRef = EA85B7A92A6860370096E097 /* TestObserver.swift */; };
```

Comment:
are these expected, or are leftovers from adding/removing the file multiple times? 

(if it's the latter, which _always_ happens to me, i usually discard all changes in the `./project.pbxproj` before submitting the changes and rebuild the project so the references will be added to the right place in the file)

Reply:
Good catch!! 👏 
I did remove one of them in [`937cae8`](https://github.com/wordpress-mobile/WordPress-iOS/pull/21148/commits/937cae8b25cd94ed5923a7bdb0ad1705a1a0c3df), the other two are related to JetpackUI Tests and WordPressUITests build phase references.